app/api/dds/v1/children_api.rb

```diff
@@ -14,6 +14,9 @@ class ChildrenAPI < Grape::API
       end
       params do
         optional :name_contains, type: String, desc: 'list children whose name contains this string'
+        optional :exclude_response_fields, type: Array[String],
```

Comment:
you should run launch_application and view the /apiexplorer to see if/how this ends up in swagger.

Reply:
Na, I don't feel like messing with `launch_application`. If after deploying to ua_test we find that it's not usable in swagger, I'll look into other solutions.